### PR TITLE
added readme to project

### PR DESCRIPTION
**Pull request by chris-chen-creates**
Opening new PR for addition of a README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
 
-### How to run
\ No newline at end of file
+## How to run
+1. Open a terminal window
+2. Navigate to the Archilator project
+3. Within terminal, use the command:
+   
+        npm start 
+4. Type in some math problems using any of the following characters: `0-9 + - * / . ( ) ^`
```
Comment (angles-n-daemons) [inline README.md]: nit: `Type in some math problems` -> `Enter an arithmetic equation into the prompt`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+5. Press **ENTER**
+6. The math problem output will be displayed on the next line
```
Comment (angles-n-daemons) [inline README.md]: nit: `math problem output` -> `solution`

File: README.md [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
+A simple terminal-based calculator 🧮
 
-### Dependencies
+## Dependencies
+  - [node.js](nodejs.org)
+  - [npm](https://www.npmjs.com/)
 
-### Setup
+## Setup  
+1. Download the .zip file or clone the repo using:
```
Comment (angles-n-daemons) [inline README.md]: nit: `using the github command line tool`

**Review by angles-n-daemons — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,9 +1,25 @@
-# ARCHILATOR
+# 🅰🆁🅲🅷🅸🅻🅰🆃🅾🆁
 
-A simple terminal-based calculator
+A simple terminal-based calculator 🧮
 
-### Dependencies
+## Dependencies
+  - [node.js](nodejs.org)
```
Comment (angles-n-daemons) [inline README.md]: Could you specify which version of node you're running?

Comment (chris-chen-creates) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (chris-chen-creates) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (chris-chen-creates) [reply README.md]: changed

File: README.md (hunk 1 above)
Comment (chris-chen-creates) [reply README.md]: changed

**Pull request merged**